README.md

```diff
@@ -482,14 +488,77 @@ print(stat.size)
 
 ## Promote Docker image
 Promotes a Docker image in a registry to another registry.
-```
+```python
 from artifactory import ArtifactoryPath
 
 path = ArtifactoryPath("http://example.com/artifactory")
 
 path.promote_docker_image("docker-staging", "docker-prod", "my-application", "0.5.1")
 ```
 
+## Builds
+### Get all builds
+Since: 2.2.0  
```

Comment:
It can confuse library's users - the version here is about Artifactory, not the library.

I'm not sure that we should copy-paste documentation from Artifactory's documentation.

I'd left only `Description` section across all docs, without the word `Description`

Reply:
done